src/Apps/Order/Routes/__tests__/Utils/OrderAppTestPage.tsx

```diff
@@ -108,8 +108,18 @@ export class OrderAppTestPage extends RootTestPage {
     await this.update()
   }
 
-  async selectPaymentMethod(option: number) {
-    this.find(BorderedRadio).at(option).simulate("click")
+  async selectPaymentMethod(paymentMethod: string) {
+    switch (paymentMethod) {
+      case "CreditCard":
+        this.find("label[data-test-id='credit-card']").simulate("click")
+        break
+      case "WireTransfer":
+        this.find("label[data-test-id='wire-transfer']").simulate("click")
+        break
+      case "USBankAccount":
+        this.find("label[data-test-id='us-bank-account']").simulate("click")
+        break
+    }
```

Comment:
**suggestion (non-blocking):** This method is a big improvement over the previous one because now the test code is readable and the parameter isn't cryptic.

I think we can make a small improvement on the parameter (which needs to be a specific string) by breaking this up into 3 methods: `selectCreditCard()`, `selectWireTransfer()`, `selectUSBankAccount()`. That's more methods, but each one will be small and simple, so it's easy to read and easy to use.